### ISSUE TITLE
Badge: Black border on grey

### ISSUE THREAD

**Issue by EldRoos** [seb-oss/green#940]
### Does the component already exist

- [X] I confirm that I have checked if the component already exists in any of the Green libraries

### Description

Badge have black border on light grey color.
https://designlibrary.sebank.se/components/component-badge

### Who will benefit from this feature?

a11y

**Comment by EldRoos** [comment]
Design changed so code is now same. Closing.
